ProjectShop/UserApp/models.py

```diff
@@ -74,8 +75,10 @@ def has_perm(self, perm, obj=None):
         return self.role
 
     def has_module_perms(self, app_label):
+        if app_label == 'token_blacklist':
+            return self.is_superuser
         return self.role
 
     @property
     def is_staff(self) -> int:
-        return self.role
+        return not self.is_bot and self.role
```

Comment:
советую not оборачивать в скобки, будет удобнее
```suggestion
        return (not self.is_bot) and self.role
```